pkg/apis/meta/v1/deployment/deployments.go

```diff
@@ -0,0 +1,82 @@
+package deployment
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+
+	"k8s.io/client-go/kubernetes"
+)
+
+type MissingDeployment struct {
+	Name                           string
+	ReadyReplicatError             error
```

Comment:
did you mean `ReadyReplicaError` or `ReadyReplicasetError` ? Extra `t` is there

Reply:
I mean ReadyReplicat error because it is what we are testing https://github.com/open-cluster-management/library-go/blob/78d5eaf0bbafed5b18cb12701e8d97600692b973/pkg/apis/meta/v1/deployment/deployments.go#L63 but there is a type... should be ReadyReplicasError and not ReplicaT